glean-core/src/lib.rs

```diff
@@ -929,12 +938,12 @@ impl Glean {
     /// This functions generates a baseline and an events ping with reason
     /// `inactive` and then clears the dirty bit.
     pub fn handle_client_inactive(&mut self) {
-        if let Err(err) = self.internal_pings.baseline.submit(self, Some("inactive")) {
-            log::warn!("Failed to submit baseline ping on inactive: {}", err);
+        if !self.internal_pings.baseline.submit(self, Some("inactive")) {
+            log::info!("baseline ping not submitted on inactive");
         }
 
-        if let Err(err) = self.internal_pings.events.submit(self, Some("inactive")) {
-            log::warn!("Failed to submit events ping on inactive: {}", err);
+        if !self.internal_pings.events.submit(self, Some("inactive")) {
+            log::info!("events ping not submitted on inactive");
```

Comment:
I wonder if it should still be louder than `info!`, though, since it should be impossible. Thoughts? Maybe it needs instrumentation/error stream.

Reply:
Not having an events ping on inactive is normal behavior though, if no events have been recorded nothing will be submitted.